server/game/cards/08-MotC/CourteousScheming.js

```diff
@@ -0,0 +1,28 @@
+const DrawCard = require('../../drawcard.js');
+const { Durations, DuelTypes } = require('../../Constants');
+
+class CourteousScheming extends DrawCard {
+    setupCardAbilities(ability) {
+        this.action({
+            title: 'Initiate a political duel',
+            condition: () =>
+                this.game.currentConflict &&
+                this.game.currentConflict.conflictType === 'political',
+            initiateDuel: () => ({
+                type: DuelTypes.Political,
+                opponentChoosesDuelTarget: true,
+                gameAction: duel =>
+                    duel.winner &&
+                    ability.actions.playerLastingEffect({
+                        targetController: duel.winner.controller,
```

Comment:
This should be a Players constant I think, so I'm surprised this is passing tests, as it shouldn't work on the opponent like this!

Reply:
thanks! i think Courtly Challenger is also doing

```
gameAction: duel => AbilityDsl.actions.draw({
    amount: 2,
    target: duel.winner.controller
})
```

Reply:
Courtly challenger isn't applying a lasting effect.  Player lasting effects work in an unintuitive way due to legacy code in the effect engine.  I should get round to refactoring it to work the way everyone expects it to work, but yeah... time